src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java

```diff
@@ -221,13 +197,16 @@ else if (participantType == ParticipantType.edge)
 
             ops.setCapability(CapabilityType.LOGGING_PREFS, logPrefs);
 
-            if (!Boolean.getBoolean(PROP_DISABLE_NOSANBOX))
+            // FIXME This is hard to understand. The property is "to disable",
```

Comment:
I completely agree with the comment. But why are we adding confusing method names on top of the confusing property name? We can name the method e.g. `sandboxEnabled` and implement the logic there.
And longer term we can deprecate the old property, add a new one, and handle both in `sandboxEnabled`.

Reply:
I'm looking at the PR one commit at a time, so I might have missed changes in later commits.

Reply:
So the option we pass to chrome is 'no-sandbox', and before that the static variable was DISABLE_NOSANBOX_PARAM, which is better when reading than PROP_DISABLE_NOSANBOX, but this can be only me. I know that probably it became PROP_DISABLE_NOSANBOX to keep consistency with other props.

Reply:
I think this is easier to read
```if (!options.enableSandbox())
{
    ops.addArguments("no-sandbox");
}```

Compared to `if (!options.isDisableNoSandbox())`

Or better: `if(options.disableSandbox())`

Reply:
I will go with "disableSandbox" which was your last "or better" comment assuming it's the preferred way.